pom.xml

```diff
@@ -104,6 +104,12 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>2.6.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.mvel</groupId>
+            <artifactId>mvel2</artifactId>
```

Comment:
why do you need this dependency?

Reply:
i maid custom @FieldMatch validation and use this dependency there. but i see you've simplified the validation method and here i remove it now. Thank you!